bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -142,6 +144,13 @@ private static void performWarmup() {
     connectionStartupExecutor.execute(new Runnable() {
       @Override
       public void run() {
+        // The first invocation of enableTracing does a reflection lookup which may be expensive.
+        enableTracing(null);
+      }
```

Comment:
This is a bit confusing, I think it would be better to pay the cost of reflection than to try to prefetch the reflection methods.